packages/shared/lib/core/profile/actions/profiles/checkAndMigrateProfiles.ts

```diff
@@ -167,7 +168,7 @@ function persistedProfileMigrationToV10(existingProfile: IPersistedProfile): voi
 
 function persistedProfileMigrationToV11(existingProfile: IPersistedProfile): void {
     const network = NETWORK?.[existingProfile?.networkProtocol]?.[existingProfile?.networkType]
-    existingProfile.network = network
+    existingProfile.network = deepCopy(network)
```

Comment:
I recently learned that we can use the `structuredClone` algorithm directly, which is quite a bit faster than the JSON-based approach in this `deepCopy` function.